app/views/devise/confirmations/new.html.erb

```diff
@@ -1,17 +1,18 @@
-<h2><%= t('.page_title') %></h2>
-
-<%= form_for(resource, as: resource_name, url: confirmation_path(resource_name), html: { method: :post }) do |f| %>
-  <%= render "devise/shared/error_messages", resource: resource %>
-
-  <div class="field">
-    <%= f.label :email %><br />
-    <%= f.email_field :email, autofocus: true, autocomplete: t('.email'),
-                      value: (resource.pending_reconfirmation? ? resource.unconfirmed_email : resource.email) %>
-  </div>
-
-  <div class="actions">
-    <%= f.submit t(".resend_confirmation_instructions") %>
+<div class="form-group col">
+  <div class="form-group justify-content-md-center">
+    <h2 class="form-group offset-3 col-6"><%= t('.page_title') %></h2>
```

Comment:
It's not a good idea to use `form-group` class in a three-block structure
Also, move `offset-3 col-6` to the parent tag, you don't need to use it each time.

Reply:
I will move it.